### PR TITLE
eslint config ovh

### PR DESCRIPTION
**Pull request by frenautvh**

### CODE REVIEW

File: packages/oui-action-menu/src/index.spec.js [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+            expect(ulElement).toBeTruthy();
+        });
 
-    it('should display a button item', () => {
-      const clickSpy = jasmine.createSpy('spy')
-      const element = TestUtils.compileTemplate(`
+        it("should display a button item", () => {
+            const clickSpy = jasmine.createSpy("spy");
+            const element = TestUtils.compileTemplate(`
         <oui-action-menu>
```
Comment (jimmyfortinx) [inline packages/oui-action-menu/src/index.spec.js]: indentation

File: packages/oui-action-menu/src/index.spec.js [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
           <oui-action-menu-item text="Action 1"
             on-click="$ctrl.clickHandler()"></oui-action-menu-item>
         </oui-action-menu>`,
-        {
-          clickHandler: clickSpy
-        })
+                                                      {
```
Comment (jimmyfortinx) [inline packages/oui-action-menu/src/index.spec.js]: indentation

File: packages/oui-action-menu/src/index.spec.js [hunk 3]
```diff
@@ -1,55 +1,55 @@
...
+            angular.element(buttonElement).triggerHandler("click");
+            expect(clickSpy).toHaveBeenCalled();
+            expect(clickSpy.calls.count()).toEqual(1);
+        });
 
-    it('should display a link item', () => {
-      const element = TestUtils.compileTemplate(`
+        it("should display a link item", () => {
+            const element = TestUtils.compileTemplate(`
         <oui-action-menu>
```
Comment (jimmyfortinx) [inline packages/oui-action-menu/src/index.spec.js]: indentation

File: packages/oui-dropdown/src/index.spec.js [hunk 4]
```diff
@@ -1,158 +1,158 @@
...
+        $document = _$document_;
+    }));
 
-  describe('Component', () => {
-    it('should display the default trigger', () => {
-      const element = TestUtils.compileTemplate(`
+    describe("Component", () => {
+        it("should display the default trigger", () => {
+            const element = TestUtils.compileTemplate(`
         <oui-dropdown>
```
Comment (jimmyfortinx) [inline packages/oui-dropdown/src/index.spec.js]: indentation and for others in this file

File: packages/oui-action-menu/src/index.spec.js (hunk 1 above)
Comment (jimmyfortinx) [reply packages/oui-action-menu/src/index.spec.js]: Done

File: packages/oui-action-menu/src/index.spec.js (hunk 2 above)
Comment (jimmyfortinx) [reply packages/oui-action-menu/src/index.spec.js]: Done

File: packages/oui-action-menu/src/index.spec.js (hunk 3 above)
Comment (jimmyfortinx) [reply packages/oui-action-menu/src/index.spec.js]: Done

File: packages/oui-dropdown/src/index.spec.js (hunk 4 above)
Comment (jimmyfortinx) [reply packages/oui-dropdown/src/index.spec.js]: Done

File: .eslintrc.js [hunk 5]
```diff
@@ -1,24 +1,41 @@
...
+        "node": true,
+        "jasmine": true,
+        "es6": true
+    },
+    "globals": {
+        "angular": true,
+        "inject": true
+    },
+    "extends": "ovh",
+    "rules": {
```
Comment (jleveugle) [inline .eslintrc.js]: Why do you need to update theses rules ? 2 solutions : 
+ Updating theses rules is legit, and we need to update the OVH rules
+ Else, we should follow the existing rules :)

But please, don't create a different `.eslintrc.js`. The goal is to homogenise :)

Comment (tremblaymath) [reply .eslintrc.js]: OVH rules will be updated and this custom config removed at the end of V2

**Pull request merged**